### ISSUE TITLE
xsk_fd=0 bug

### ISSUE THREAD

**Issue by ripatel-fd** [firedancer-io/firedancer#1105]
Multiple operators have reported a bug where Firedancer networking ceases to function entirely. Associated log output

```
net  src/tango/xdp/fd_xsk.c(846): xsk sendto failed xsk_fd=0 (6-ENXIO-no such device or address)
```

**Comment by ripatel-fd** [comment]
@mmcgee-jump I've put this as prio high, please feel free to change.

**Comment by ripatel-fd** [comment]
@mmcgee-jump Has this been fixed?

**Comment by mmcgee-jump** [comment]
No, we did not get further diagnostics from the user to debug it. I trawled kernel code and could not figure out why this would happen. It is likely related to running in a container, but need confirmation from user.

**Comment by mmcgee-jump** [comment]
Probably fixed by #1294

**Comment by mmcgee-jump** [comment]
Not fixed
